tests/test_stars.py

```diff
@@ -133,15 +263,16 @@ def test_de_json_star_transaction_localization(self, star_transaction, tz_bot, b
         assert st_bot.date.tzinfo == UTC
         assert st_offset == tz_bot_offset
 
-    def test_to_dict(self, star_transaction):
+    def test_to_dict(self):
+        st = star_transaction()
```

Comment:
did you mean to use this as fixuter instead, i.e. pass it as argument? same in the other tests

Reply:
I had done that before but I actually wanted to use the return value in `TestStarTransactionsBase`, so I kept it as a function

Reply:
Mh, I see. Might something like this be of use? https://stackoverflow.com/a/63047695/10606962
I'm okay with keeping as is, but it's at least something that future-us will likely stumble over if we have re-touch tests.